packages/twenty-front/src/pages/settings/accounts/SettingsAccountsEmailsInboxSettings.tsx

```diff
@@ -19,6 +20,10 @@ export const SettingsAccountsEmailsInboxSettings = () => {
     if (!account) navigate(AppPath.NotFound);
   }, [account, navigate]);
 
+  if (!account) return null;
```

Comment:
It seems redundant with the useEffect above? If not, should we redirect to NotFound as well?
cc @thaisguigon

Reply:
It allows TypeScript to know that account is defined below this if statement.